pkg/cache/clusterqueue.go

```diff
@@ -306,6 +328,16 @@ func (c *clusterQueue) inactiveReason() (string, string) {
 	return kueue.ClusterQueueActiveReasonReady, "Can admit new workloads"
 }
 
+func (c *clusterQueue) isTASViolated() bool {
+	if !features.Enabled(features.TopologyAwareScheduling) || len(c.tasFlavors) == 0 {
+		return false
+	}
+	return c.HasParent() ||
+		c.Preemption.WithinClusterQueue != kueue.PreemptionPolicyNever ||
+		len(c.multiKueueAdmissionChecks) > 0 ||
+		len(c.provisioningAdmissionChecks) > 0
+}
```

Comment:
Will we support these combinations between TAS and other features in the future (Beta or GA)?

Reply:
It is in graduation criteria for GA, but if we get requests from users I think we can accelerate support for some of the features.

Reply:
Oh, I missed the GA criteria. SGTM.